plugins/snapshots/erofs/erofs_linux.go

```diff
@@ -355,6 +355,31 @@ func (s *snapshotter) View(ctx context.Context, key, parent string, opts ...snap
 	return s.createSnapshot(ctx, snapshots.KindView, key, parent, opts)
 }
 
+func setImmutable(path string, enable bool) error {
+	//nolint:revive	// silence "don't use ALL_CAPS in Go names; use CamelCase"
+	const (
+		FS_IMMUTABLE_FL = 0x10
```

Comment:
Just for context, where did you find the bitmask for this flag? I couldn't find this in the [ioctl_flags(2) docs](https://man7.org/linux/man-pages/man2/ioctl_iflags.2.html).

Reply:
really?